### PR TITLE
Tomcat: initial integration

### PR DESCRIPTION
**Pull request by henryrneh**
Initial integration of Apache Tomcat.

### CODE REVIEW

**Review by oliverchang — APPROVED** [review]

File: projects/tomcat/HttpParserFuzzer.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+import org.apache.tomcat.util.http.parser.HttpParser;
+
+public class HttpParserFuzzer {
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
```
Comment (oliverchang) [inline projects/tomcat/HttpParserFuzzer.java:6]: Ideally we'd like to see some more interesting targets :)

Comment (0roman) [reply projects/tomcat/HttpParserFuzzer.java:6]: @oliverchang there will be more tests integrated in the next week.

Comment (henryrneh) [reply projects/tomcat/HttpParserFuzzer.java:6]: Thank you for your feedback!

Comment (oliverchang) [reply projects/tomcat/HttpParserFuzzer.java:6]: OK, looking forward to it! Merging this for now.

**Pull request merged**